scipy-1.0/paper.tex

```diff
@@ -298,25 +298,26 @@ \subsubsection*{LowLevelCallable}
-These wrappers are available in the modules \texttt{scipy.{\allowbreak}linalg.{\allowbreak}cython\_blas}, \texttt{scipy.{\allowbreak}linalg.{\allowbreak}cython\_lapack}, and \texttt{scipy.{\allowbreak}special.{\allowbreak}cython\_special} respectively.
-The low-level interfaces for Cython can also be used by third party libraries to get direct access to the functions in these libraries while avoiding the overhead of Python function calls{\textemdash}this can give performance gains of one or two orders of magnitude for many use cases.
-Cython's cimport machinery allows Cython based modules to share C-level objects with other Cython modules without requiring that the third-party extensions link against any additional libraries.
-Function pointers to wrappers for library routines can also be shared in this way, so that Cython modules using the interface provided by SciPy no longer have to worry about finding and linking against the original library.
-This lets other extensions avoid the complexity of finding and using the correct libraries on the various possible Fortran compiler ABI and OS combinations.
-The low-level Cython wrappers provide an OS and Fortran compiler independent interface for the libraries they wrap.
-
-To help with both correctness and ease of maintenance, most of these low-level Cython wrappers are generated automatically.
+\subsection*{Cython bindings for BLAS, LAPACK, and special}
+
+SciPy includes Cython\cite{behnel2011cython} wrappers for many BLAS and LAPACK\cite{LAPACK} routines and for the special functions provided in the \texttt{scipy.{\allowbreak}special} submodule.
+These Cython wrappers are available in the modules \texttt{scipy.{\allowbreak}linalg.{\allowbreak}cython\_blas}, \texttt{scipy.{\allowbreak}linalg.{\allowbreak}cython\_lapack}, and \texttt{scipy.{\allowbreak}special.{\allowbreak}cython\_special} respectively.
+When writing algorithms in Cython, it is typically more efficient to call directly into the libraries SciPy wraps rather than indirectly, using SciPy's Python APIs.
+These low-level interfaces for Cython can also be used outside of the SciPy codebase to gain access to the functions in the wrapped libraries while avoiding the overhead of Python function calls.
+This can give performance gains of one or two orders of magnitude for many use cases.
+
+Developers can also use the low-level Cython interfaces without linking against the wrapped libraries\cite{blas-lapack-wrappers-scipy-2015}.
+This lets other extensions avoid the complexity of finding and using the correct libraries.
+Avoiding this complexity is especially important when wrapping libraries written in Fortran.
+Not only can these low-level wrappers can be used without a Fortran compiler, they can also be used without having to handle all the different Fortran compiler ABI's and name mangling schemes.
```

Comment:
grammar thingy here, "can ... can be used"

Reply:
Fixed. Thank you!